06_Userspace/03_Handling_Interrupts.md

```diff
@@ -55,11 +55,12 @@ The layout of the TSS system descriptor is broken down below in the following ta
 
 | Bits  | Should Be Set To | Description                         |
 |--------|------------------|-------------------------------------|
-| 15:0   | 0xFFFF           | Represents the limit field for this segment. Ignored in long mode, but best set to max value in case we support compatibility mode in the future. |
+| 15:0   | 0xFFFF           | Represents the limit field for this segment. |
 | 31:16 | TSS address bits 15:0 | Contains the lowest 16 bits of the tss address. |
 | 39:32  | TSS address bits 23:16 | Contains the next 8 bits of the tss address. |
-| 47:40  | 0b10001001 | Sets the type of GDT descriptor, it's DPL (bits 45:46) to 0, marks it as present (bit 47). The rest of this magic value indicates it's a valid TSS descriptor. If curious as to how this value was created, see the  manual or our section about the GDT.|
-| 55:48  | 0b10000 | Additional fields for the TSS entry. This bit means the TSS is `available`, it's generally unused in long mode, but has some side effects if compatibility mode is enabled. |
+| 47:40  | 0b10001001 | Sets the type of GDT descriptor, it's DPL (bits 45:46) to 0, marks it as present (bit 47). Bit 44 (S) along with bits 40 to 43 indicate the type of descriptor. If curious as to how this value was created, see the  intel SDM manual or our section about the GDT.|
```

Comment:
not sure i got what change you are requesting. What code snipped you mean? 

The purpose is just try to explaine what the bits are.

Reply:
If possible, try to add a hyper link, very similar to what you use in Microsodt Word, but that process is not gaurenteed, because the file name, binary, text, properties may not be sufficient to supliment a hyper text technique. But if possible, try to add a hyper link of the specified environment to the text file and beta test with that. It is confusing, I know, but hey, atleast the customers don't need to manualy go and check the specified environment...

Reply:
customers? This is an open source book.
If you know of a way to hyperlink into a pdf on someone else's server then feel free to suggest the code (not pseudocode!), but otherwise this makes no sense.

Reply:
------------------------------------------------------------------------------------------

> customers? This is an open source book. If you know of a way to hyperlink into a pdf on someone else's server then feel free to suggest the code (not pseudocode!), but otherwise this makes no sense.

------------------------------------------------------------------------------------------

Sorry for using incorrect grammar **DeanoBurrito**! I meant:

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

 _the viewers/reviewers don't need to manually go and check the specified environment..._

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Yes, I do know a way to hyperlink into a PDF on someone else's server. I am pleased to share the code! You link to a PDF file the same way you would link to an HTML file: using the "a" anchor tag and the URL address of the PDF. The code you might be asking is:

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

< a href="#Specify_a_Hyperlink_Target_href">If curious as to how this value was created, see the  intel SDM manual or our section about the GDT.|</ a>

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

But make sure  to remove the spaces in **< a href="#Specify_a_Hyperlink_Target_href">** and **</ a>**, because GitHub renders this comment as an actual code and produces a link which leads into a webpage, which is not good. Git hub prevents that from happening, so no worries. I sincerely Apologise to use wrong grammar and to confuse you people to use pseudocode! If this code is malfunctioning, please call me, or my team mates at https://github.com/TBTS-pvt-ltd-2011.

Have a good day pal!
Pavan- From The table of The Beta Testing Studios.pvt.ltd
11 / 10 / 2023

------------------------------------------------------------------------------------------